src/index.css

```diff
@@ -9,9 +9,23 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  @apply text-lg;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+h1 {
+  @apply text-3xl;
 }
+
+h2 {
+  @apply text-2xl;
+  @apply text-center;
+  @apply mb-4;
+}
+
+h3 {
+  @apply mb-4;
+}
+
+a {
+  @apply hover:underline;
+}
```

Comment:
I think this is a slight anti-pattern here. Just because it's going to interfere with every other page's CSS so I would lean slightly on the side of only using it on tags we need it.

Reply:
Oh I actually _do_ want it to interfere with every page. Imo this is setting a sensible default for a link in the site